client/backoff/backoff.go

```diff
@@ -64,62 +64,49 @@ func (b *Backoff) Reset() {
 // It will backoff if the function returns a retryable error.
 // Once the context is done, retries will end and the most recent error will be returned.
 // Backoff is not reset by this function.
-func (b *Backoff) Retry(ctx context.Context, f func() error) error {
+func (b *Backoff) Retry(ctx context.Context, f func() error, retryableCodes ...codes.Code) error {
```

Comment:
nit: Could rename `retryableCodes` to just `retryable`, because the type already implies it's `Codes`.

Reply:
Did you resolve this because you decided to leave it as is?

Reply:
My mistake. I renamed it in the function below, but not here.